cysync/src/pages/mainApp/sidebar/wallet/send/formStepComponents/Recipient.tsx

```diff
@@ -612,7 +640,7 @@ const Recipient: React.FC<StepComponentProps> = props => {
 
   return (
     <Root container className={root}>
-      {!isEthereum && (
+      {!isEthereum && !isNear && (
```

Comment:
Let's keep this as a positive check condition rather than a negation.
Would suggest to add a `isBitcoin` flag on top similar to the existing flags.